rust/phoenix-sdk-core/src/sdk_client_core.rs

```diff
@@ -101,24 +103,33 @@ impl SDKClientCore {
     /// Converts base units to base lots. For example if the base currency was a Widget and you wanted to
     /// convert 3 Widgets to base lots you would call sdk.base_unit_to_base_lots(3.0). This would return
     /// the number of base lots that would be equivalent to 3 Widgets.
-    pub fn base_units_to_base_lots(&self, base_units: f64) -> u64 {
-        (base_units * self.base_multiplier as f64 / self.base_lot_size as f64) as u64
+    pub fn raw_base_units_to_base_lots(&self, raw_base_units: f64) -> u64 {
```

Comment:
fix doc

Reply:
I might just be missing it but what is wrong here? Perhaps "Converts raw base units" ?

Reply:
yes, and the function name in the doc doesn't match the actual function name

Reply:
Ah gotcha thanks! Changed in the latest commit